uvicorn/protocols/http/httptools_impl.py

```diff
@@ -21,12 +35,18 @@
     get_remote_addr,
     is_ssl,
 )
+from uvicorn.server import ServerState
+
+if sys.version_info < (3, 8):  # pragma: py-gte-38
+    from typing_extensions import Literal
+else:  # pragma: py-lt-38
+    from typing import Literal
```

Comment:
do we have a `_compat.py` module or something where this sorta boilerplate can go?

Reply:
No